### ISSUE TITLE
no logo in dashboard

### ISSUE THREAD

**Issue by arbin0077** [AlashShrestha/GroupE_G6#17]
Please add logo of the product in dashboard

**Comment by Dikesh9811** [comment]
Yes, this has been solved by adding div class "logo" and image source

![image](https://github.com/AlashShrestha/GroupE_G6/assets/127932182/a25e2500-0d1f-4a38-973c-7e52716910cb)
